index.js

```diff
@@ -115,10 +115,16 @@ class SignaturePad extends Component {
 
   };
 
+  onMessage = (event) => {
+    var base64DataUrl = JSON.parse(event.nativeEvent.data);
+    this._bridged_finishedStroke(base64DataUrl);
```

Comment:
Ok. Is it because you want `base64DataUrl` to be immutable, or is there something wrong with this way.
We have used it in our app for a year this way and it works.

Reply:
@freefony Well, I've just checked it myself. Indeed, it works as you said. Please, disregard my previous comments.